From 7a09c41c0244b751101ada26f0353451a5a13b23 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Sat, 12 Dec 2015 16:01:01 +0100
Subject: [PATCH 1/5] Fix tab mixing

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 src/Quest.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Quest.py b/src/Quest.py
index 9d101ce..f84a8b4 100644
--- a/src/Quest.py
+++ b/src/Quest.py
@@ -458,13 +458,13 @@ def demo():
         tTest=tTest+random.choice([-0.1,0,0.1])
 
         # Simulate a trial
-	timeSplit=time.time(); # omit simulation and printing from reported time/trial.
+        timeSplit=time.time(); # omit simulation and printing from reported time/trial.
         response=q.simulate(tTest,tActual)
         print 'Trial %3d at %4.1f is %s'%(k+1,tTest,wrongRight[int(response)])
-	timeZero=timeZero+time.time()-timeSplit;
+        timeZero=timeZero+time.time()-timeSplit;
         
         # Update the pdf
-	q.update(tTest,response);
+        q.update(tTest,response);
 
     # Print results of timing.
     print '%.0f ms/trial'%(1000*(time.time()-timeZero)/trialsDesired)
-- 
2.6.4

